overrides.nix

```diff
@@ -1839,4 +1839,16 @@ self: super:
       '';
     });
 
+  marisa-trie = super.marisa-trie.overridePythonAttrs (
+    old: {
+      buildInputs = (old.buildInputs or [ ]) ++ [ self.pytest-runner ];
+    }
+  );
+
+  ua-parser = super.ua-parser.overridePythonAttrs (
+    old: {
+      buildInputs = (old.buildInputs or [ ]) ++ [ self.pyyaml ];
```

Comment:
Are you sure this one shouldn't be propagated?

Reply:
Yes you're right. I did not check where pyyaml was used and it is used outside of the setup.py as well.
I'm updating the PR